pkg/ui/src/views/cluster/containers/nodeGraphs/summaryBar.tsx

```diff
@@ -86,8 +93,11 @@ export default function(props: ClusterSummaryProps) {
                                         usable storage capacity across all nodes.`} />
         </SummaryStat>
         <SummaryStat title="Unavailable ranges" value={props.nodesSummary.nodeSums.unavailableRanges} />
-        <SummaryMetricStat id="qps" title="Queries per second" format={formatOnePlace} >
-          <Metric sources={props.nodeSources} name="cr.node.sql.query.count" title="Queries/Sec" nonNegativeRate />
+        <SummaryMetricStat id="qps" title="Queries per second" format={formatOnePlace} aggregator={TimeSeriesQueryAggregator.SUM} >
```

Comment:
Fyi, updated component here to use the exact same `<Metric/>` components that Node Map (`<QpsSparkline/>`) and Time Series (`dashboards\overview.tsx`) use.